examples/README.md

```diff
@@ -69,34 +69,78 @@ Coming soon!
 **Coming soon!**
 -->
 
+## Distributed training and mixed precision
+
+All the PyTorch scripts mentioned above work out of the box with distributed training and mixed precision, thanks to
+the [Trainer API](https://huggingface.co/transformers/main_classes/trainer.html). To launch one of them on 8 GPUS, use
```

Comment:
```suggestion
the [Trainer API](https://huggingface.co/transformers/main_classes/trainer.html). To launch one of them on the number of GPUS you have, use
```

or maybe `n` GPUS because below we are using `number_of_gpu_you_have` in the command